### PR TITLE
Bisapsltsau A DZ 1

### PR DESCRIPTION
**Pull request by MsWik**
DZ 1, B. Andry
Second try. It seems to have understood.

### CODE REVIEW

File: biaspaltsau_a/02.Git.Local/README.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+## 02.Git.Local
+'''
```
Comment (pluhin) [inline biaspaltsau_a/02.Git.Local/README.md:2]: Wrong github markdown

File: biaspaltsau_a/02.Git.Local/README.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+   82  git checkout dev
+   83  git merge hotfix
+   84  git merge features
+   85  git checkout features
+   86  git merge hotfix
+   87  history
+
+
+'''
+## FINAL
```
Comment (pluhin) [inline biaspaltsau_a/02.Git.Local/README.md:46]: Missing logs from branches, please add them

**Review by pluhin — CHANGES_REQUESTED** [review]

File: biaspaltsau_a/02.Git.Local/README.md (hunk 1 above)
Comment (MsWik) [reply biaspaltsau_a/02.Git.Local/README.md:2]: I do not understand what is the meaning of the error.

**Review by pluhin — COMMENTED** [review]
Please see this

https://docs.github.com/en/github/writing-on-github/creating-and-highlighting-code-blocks

**Review by pluhin — CHANGES_REQUESTED** [review]
still wrong :(

**Comment by MsWik** [comment]
add DZ_2, no SSH, HTPS only 
##############
$ git push -u origin main
kex_exchange_identification: read: Connection reset by peer
Connection reset by 140.82.113.4 port 22
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
###############

**Review by pluhin — APPROVED** [review]
Very good, approved

**Pull request merged**